lib/src/default_index/revset_engine.rs

```diff
@@ -130,8 +130,21 @@ impl<I: AsCompositeIndex> Revset for RevsetImpl<I> {
         self.entries().next().is_none()
     }
 
-    fn count(&self) -> usize {
-        self.entries().count()
+    fn size_hint(&self) -> (usize, Option<usize>) {
```

Comment:
@yuja, did you say you prefer `count_estimate()`? I don't mind either way.

Reply:
Ah, I missed this. I'll switch back to count_estimate.